### ISSUE TITLE
Invoice PDF : Text overlapse footer and pagination pushed away

### ISSUE THREAD

**Issue by arirangz** [PrestaShop/PrestaShop#23181]
#### Describe the bug

When getting an invoice with a lot of products, the products table overlapse the footer (see screenshot attached).
In addition, the pagination is too close from the edge of the page if the address is to long. The pagination can be pushed away if the address is even longer (see screenshot in comment https://github.com/PrestaShop/PrestaShop/issues/23181#issuecomment-776572565_

#### Expected behavior

Have a margin on the top of the footer and at the bottom of the footer

#### Steps to Reproduce
Open an invoice of an order with a lot of products

**Screenshots**
![image](https://user-images.githubusercontent.com/1193727/107266303-9b8e1600-6a45-11eb-8130-b59b40cd2ea0.png)

If applicable, add screenshots or screen recordings to help explain your problem.

#### Additional information

* PrestaShop version: 1.7.7.1
* PHP version: N/A

**Comment by hibatallahAouadni** [comment]
Hello @arirangz 

Thanks for your issue! I reproduced your issue with PS1.7.7.1 and PS1.7.6.7, see the attached screenshot below:

- **1.7.6.7:**

![image](https://user-images.githubusercontent.com/23308427/107280338-87074900-6a58-11eb-8af9-18d46ad0b48c.png)

- **1.7.7.1:**

![image](https://user-images.githubusercontent.com/23308427/107280442-a9996200-6a58-11eb-854e-031ffe10b786.png)

I’ll add this to the debug backlog so that it’s fixed.
If you have already fixed it on your end or if you think you can do it, please do send us a pull request!

Thanks!

**Comment by NeOMakinG** [comment]
I think a backend dev could handle this as tcPDF got a setFooterMargin method

**Comment by arirangz** [comment]
hello @hibatallahAouadni 
Thanks reproducing the overlapsed text.
Did you also reproduce the pagination too close from the edge?

**Comment by hibatallahAouadni** [comment]
Hello @arirangz 

Thanks to mention the pagination :clap: 
Well, that another bug and it's not related to invoice content but to address length, see the attached screen shot below:

Long address => no pagination

![image](https://user-images.githubusercontent.com/23308427/107491107-039a4480-6b8b-11eb-9287-038f62b18e9c.png)

Short address => pagination

![image](https://user-images.githubusercontent.com/23308427/107491178-16147e00-6b8b-11eb-9e95-14b969f09bb9.png)

Thanks!

**Comment by arirangz** [comment]
thanks @hibatallahAouadni 
So should I rename the issue title to add the pagination issue?

**Comment by hibatallahAouadni** [comment]
@arirangz yes please and add some details about the pagination in your issue description. Thanks :slightly_smiling_face:

**Comment by arirangz** [comment]
@hibatallahAouadni Ok I did it. Maybe you should then update the Epic? thanks :)
